tools/runners/run-mongo.js

```diff
@@ -213,8 +213,7 @@ if (process.platform === 'win32') {
             var foundPort = parseInt(m[2], 10);
             var foundPath = m[3];
 
-            if ( (! port || port === foundPort) &&
-                 (! dbDir || dbDir === foundPath)) {
+            if (! dbDir || dbDir === foundPath) {
```

Comment:
When I was debugging this issue, I didn't recall running into this issue, but yes, the original check raises some question and as @benjamn says:

> Since we're still checking the --dbPath, this search won't return any mongod processes that don't belong to this app.